cli-installation/README.md

```diff
@@ -2,13 +2,16 @@
 
 ## Prerequisites
 
-If you are working on this lab as part of the Cloudify official training course, you will be receiving
-the following from the instructor:
+To be able to perform the the following exercises you will need:
 
 * Public and private IP's of the VM on which the CLI is going to be installed
 * Private key to use in order to access that VM
 
-Copy the private key, provided by the instructor, to the file system on the machine you're going to use
+If you are working on this lab as part of the Cloudify official training course, you will be receiving
+them the following from the instructor.
```

Comment:
"Cloudify official" -> "official Cloudify"
"them the following" -> "these details"